ginkgo.Jenkinsfile

```diff
@@ -34,6 +34,25 @@ pipeline {
                 }
             }
         }
+        stage('Provision'){
```

Comment:
We aren't provisioning the VMs, so this stage shouldn't be called Provision. Please change this to "Boot VMs" or something other than "Provision".